strings/tips.txt

```diff
@@ -29,6 +29,7 @@ Most job-related exosuit clothing can fit job-related items into it. For example
 If there's something you need from another department, try asking! This game isn't singleplayer and you'd be surprised what you can get accomplished together!
 You'll quickly lose your interest in the game if you play to win and kill. If you find yourself doing this, take a step back and talk to people - it's a much better experience!
 Don't be afraid to ask for help, whether from your peers or from mentors.
+If someone is in critical condition, you can interact with them to start performing CPR. This will heal oxygen damage and remove breathloss. Additionally, if the patient is dead but still revivable, it will extend the defibrillation window for as long as CPR continues. If you or the patient is wearing a mask or you are a species that doesn't breathe the same kind of air as the patient, it will be less effective.
```

Comment:
A lot of players who this tip will be useful to probably don't know what `breathloss` is. I would either remove that from the tip or reword it in a way that indicates its gameplay impact to the standard player.

Reply:
Rephrased.